docs/linux-packaging.md

```diff
@@ -0,0 +1,215 @@
+# Linux packaging
+
+## Package distribution
+
+We use different channels to build and distribute the Linux packages:
+
+- (older version) included in official repos since Ubuntu 19.04 and Debian Buster
+- [Launchpad repo](https://launchpad.net/~keymanapp) for stable and beta versions
```

Comment:
The launchpad repo for beta versions is https://launchpad.net/~keymanapp/+archive/ubuntu/keyman-daily

Reply:
done